### PR TITLE
Use image table boot for dev config

### PR DESCRIPTION
**Pull request by jacobmcnamee**
Changes to support https://github.com/swift-nav/u-boot-xlnx/pull/20.
- Generate image set for `dev` config with U-Boot and FPGA images
- Merge `dev` and `prod` output folders

TODO:
- [x] Update U-Boot https://github.com/swift-nav/u-boot-xlnx/pull/20
- [x] Test

### CODE REVIEW

**Comment by anonymous-travis-swiftnav** [comment]
## v0.1-20-g1b00405
+ [Artifacts (HITL Dashboard Page)](http://hitl-dashboard.swiftnav.com/files/swiftnav-artifacts-pull-requests/piksi_buildroot/v0.1-20-g1b00405/)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v0.1-20-g1b00405/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v0.1-20-g1b00405

**Comment by anonymous-travis-swiftnav** [comment]
## v0.1-21-g3245e23
+ [Artifacts (HITL Dashboard Page)](http://hitl-dashboard.swiftnav.com/files/swiftnav-artifacts-pull-requests/piksi_buildroot/v0.1-21-g3245e23/)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v0.1-21-g3245e23/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v0.1-21-g3245e23

**Comment by anonymous-travis-swiftnav** [comment]
## v0.1-21-ga4ac44d
+ [Artifacts (HITL Dashboard Page)](http://hitl-dashboard.swiftnav.com/files/swiftnav-artifacts-pull-requests/piksi_buildroot/v0.1-21-ga4ac44d/)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v0.1-21-ga4ac44d/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v0.1-21-ga4ac44d

File: board/piksiv3/post_image.sh [hunk 1]
```diff
@@ -1,47 +1,62 @@
...
-generate_dev $HW_CONFIG
+mkdir -p $OUTPUT_DIR
+
+cp $UBOOT_PROD_DIR/tpl/boot.bin $OUTPUT_DIR
+cp $BINARIES_DIR/uImage.${CFG} $OUTPUT_DIR
+
+if [ -e $FIRMWARE_DIR/piksi_fpga.bit ]; then
+  generate_dev
+else
+  echo "*** NO FIRMWARE FILES FOUND, NOT BUILDING DEVELOPMENT IMAGE ***"
```
Comment (fnoble) [inline board/piksiv3/post_image.sh]: We can simplify this logic now that we require firmware images for both prod and dev. Or if you want to be able to build dev with only an FPGA image then clarify this message to state `NO FPGA IMAGE FOUND`.

**Comment by fnoble** [comment]
Other than the minor comment above, looks good. Excited to have this merged in! :shipit:

**Comment by anonymous-travis-swiftnav** [comment]
## v0.1-21-g9d166e7
+ [Artifacts (HITL Dashboard Page)](http://hitl-dashboard.swiftnav.com/files/swiftnav-artifacts-pull-requests/piksi_buildroot/v0.1-21-g9d166e7/)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v0.1-21-g9d166e7/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v0.1-21-g9d166e7

**Pull request merged**